bin/load-spark-env.sh

```diff
@@ -23,7 +23,7 @@
 
 # Figure out where Spark is installed
 if [ -z "${SPARK_HOME}" ]; then
-  export SPARK_HOME="$(cd "`dirname "$0"`"/..; pwd)"
+  source `dirname $0`/find-spark-home
```

Comment:
Let's also apply the same fix for Shellcheck complaints here and at all other occurrences of this line.